BeVolt/BSP/Simulator/DataGeneration/simulate.py

```diff
@@ -53,16 +56,26 @@ def configure():
     # Get configuration settings
     global state, mode
     print("Welcome to the BPS Simulator")
-    print("Would you like to simulate 'charging' or 'discharging'?")
+    print("Would you like to simulate 'charging', 'discharging', or 'CAN'?")
```

Comment:
I would keep CAN out of this section since 'charging' and 'discharging' are states that need to be set. Instead, you could add the CAN option to a new menu option during the KeyboardInterrupt. That way, when the user enters CTRL+C during execution, they can then have the option to simulate a CAN message. We should be able to do so while also simulating a charging/discharging battery.